### DISCUSSION TITLE
Error compiling Allegro with LAMMPS in google colab tutorial

### DISCUSSION THREAD

**Discussion by Khan-Jnu** [mir-group/pair_nequip_allegro#47] [General]
Dear Developers, 
I am trying to run the google colab tutorial and fund the that it is not compiling with the LAMMPS. I have attached the error message here. If anyone could give leads that would be very helpful. I am getting the same error in my local machine as well. 

/content/lammps/src/compute_allegro.cpp: In instantiation of ‘void LAMMPS_NS::ComputeAllegro<peratom>::compute_peratom() [with int peratom = 0]’:
/content/lammps/src/compute_allegro.cpp:191:18:   required from here
/content/lammps/src/compute_allegro.cpp:157:33: error: no matching function for call to ‘LAMMPS_NS::Comm::reverse_comm(LAMMPS_NS::ComputeAllegro<0>*)’
  157 |   if (newton) comm->reverse_comm(this);
      |               ~~~~~~~~~~~~~~~~~~^~~~~~
In file included from /content/lammps/src/compute_allegro.cpp:20:
/content/lammps/src/comm.h:79:16: note: candidate: ‘virtual void LAMMPS_NS::Comm::reverse_comm()’
   79 |   virtual void reverse_comm() = 0;                 // reverse comm of forces
      |                ^~~~~~~~~~~~
/content/lammps/src/comm.h:79:16: note:   candidate expects 0 arguments, 1 provided
/content/lammps/src/compute_allegro.cpp: In instantiation of ‘void LAMMPS_NS::ComputeAllegro<peratom>::compute_peratom() [with int peratom = 1]’:
/content/lammps/src/compute_allegro.cpp:192:18:   required from here
/content/lammps/src/compute_allegro.cpp:157:33: error: no matching function for call to ‘LAMMPS_NS::Comm::reverse_comm(LAMMPS_NS::ComputeAllegro<1>*)’
  157 |   if (newton) comm->reverse_comm(this);
      |               ~~~~~~~~~~~~~~~~~~^~~~~~
In file included from /content/lammps/src/compute_allegro.cpp:20:
/content/lammps/src/comm.h:79:16: note: candidate: ‘virtual void LAMMPS_NS::Comm::reverse_comm()’
   79 |   virtual void reverse_comm() = 0;                 // reverse comm of forces
      |                ^~~~~~~~~~~~
/content/lammps/src/comm.h:79:16: note:   candidate expects 0 arguments, 1 provided
[ 17%] Building CXX object CMakeFiles/lammps.dir/content/lammps/src/compute_cluster_atom.cpp.o
[ 17%] Building CXX object CMakeFiles/lammps.dir/content/lammps/src/compute_cna_atom.cpp.o
make[2]: *** [CMakeFiles/lammps.dir/build.make:482: CMakeFiles/lammps.dir/content/lammps/src/compute_allegro.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:161: CMakeFiles/lammps.dir/all] Error 2
make: *** [Makefile:136: all] Error 2

**Comment by Linux-cpp-lisp** [comment]
https://github.com/mir-group/pair_allegro/issues/49
